migrations/migrations.config.js

```diff
@@ -9,21 +8,16 @@ module.exports = {
         unbondingPeriod: 7,
         maxEarningsClaimsRounds: 20
     },
-    jobsManager: {
-        verificationRate: 1000,
-        verificationPeriod: 100,
-        verificationSlashingPeriod: 100,
-        failedVerificationSlashAmount: 1,
-        missedVerificationSlashAmount: 5000,
-        doubleClaimSegmentSlashAmount: 30000,
-        finderFee: 50000
+    broker: {
+        minPenaltyEscrow: (new BN(1)).mul(constants.TOKEN_UNIT).div(new BN(2)),
+        unlockPeriod: new BN(40320) // approximately 7 days worth of blocks
```

Comment:
just want to re-iterate this is an arbitrary number that we should discuss further before we deploy to testnet.

Reply:
Added a TODO in dfc5848